src/agent/IDiscCollector.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <vector>
+#include <string>
+
+class IDiscCollector
+{
+public:
+
+    virtual ~IDiscCollector() = default;
+    virtual std::vector<std::string> GetDiscsList();

```

Comment:
what will be returned in this vector?

Reply:
Nobody know now. It's work in progress :)